src/Core/Domain/Notification/QueryResult/NotificationResult.php

```diff
@@ -119,7 +125,8 @@ public function __construct(
         string $isoCode,
         string $company,
         string $status,
-        string $dateAdd
+        string $dateAdd,
+        string $customerThreadViewUrl = ''
```

Comment:
@matks reviewed the solution with @rokaszygmantas , would be nice if somebody confirm this does not break backward compatibiliy as the new property was added in order to use _legacy_link in admin_customer_threads_view.yml

Reply:
we intentionally left this `$customerThreadViewUrl` property optional in constructor, to not break BC, but maybe that's not a problem? @matks